### PR TITLE
add video support to collection / teaser

### PR DESCRIPTION
**Pull request by Phanx091**

### CODE REVIEW

File: src/atoms/Video/Video.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            autoPlay={true}
+            muted={true}
+            loop={true}
+          >
+            <source src={video.url} type="video/mp4" />
+            Your browser does not support HTML5 video.
+          </video>
+        );
+        break;
+      case 'mpr.apmcdn.org':
```
Comment (ghankerson) [inline src/atoms/Video/Video.js:24]: It looks like vimeo and apmcdn have the exact same code.  Can we find a way to combine them.  Can you do ```case 'player.vimeo.com' || 'mpr.apmcdn.org'```  or can make a function to output the same thing?

File: src/atoms/Video/__tests__/Video.test.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    video: {
+      url:
+        'https://player.vimeo.com/external/543289941.hd.mp4?s=63b6e14c286059e3c358337878bc3fe7d77d0e8d&profile_id=174',
+      caption: 'this is a video caption for vimeo',
+      background: 'true',
+      credit: { name: 'credit name', url: 'https://vimeo.com' }
+    }
+  };
+
+  const defaultPropsMprApmcdn = {
```
Comment (ghankerson) [inline src/atoms/Video/__tests__/Video.test.js:19]: Nice job on the tests!

**Comment by ghankerson** [comment]
Looks really good one small nit picky comment

File: src/atoms/Video/Video.js (hunk 1 above)
Comment (Phanx091) [reply src/atoms/Video/Video.js:24]: I tried this and it didn't work, however i will give it another go.

**Comment by ghankerson** [comment]
Ok to merge once prettier is fixed

**Pull request merged**